elements/rh-spinner/demo/rh-spinner.html

```diff
@@ -0,0 +1,31 @@
+<link rel="stylesheet" href="./demo.css"/>
+<script type="module" src="./rh-spinner.js"></script>
+
+<h1>Spinner</h1>
+    
+<pfe-band color-palette="lightest">
+    <h2>Light theme</h2>
+    <h3>Default/Large</h3>
+    <rh-spinner>Loading...</rh-spinner>
+    
+    <h3>Medium</h3>
+    <rh-spinner size="md">Loading...</rh-spinner>
+    
+    <h3>Small</h3>
+    <rh-spinner size="sm">Loading...</rh-spinner>
+</pfe-band>
+
+
+<pfe-band color-palette="darkest">
```

Comment:
Is there a way to have dark themed parent element without using `pfe-band`?

Reply:
`<rh-context-provider>`

Reply:
pfe-band won't work anyways because it provide's PFE's context, not RH's context